cmd/experimental/kjobctl/README.md

```diff
@@ -34,6 +34,24 @@ echo 'alias kjobctl="kubectl kjob"' >> ~/.bashrc
 echo 'alias kjobctl="kubectl kjob"' >> ~/.zshrc
 ```
 
+**Autocompletion:**
+
+```bash
+echo '[[ $commands[kubectl-kjob] ]] && source <(kubectl-kjob completion bash)' >> ~/.bashrc
+# Or if you are using ZSH
+echo '[[ $commands[kubectl-kjob] ]] && source <(kubectl-kjob completion zsh)' >> ~/.zshrc
```

Comment:
since the intended audience for kjobctl is composed of people that don't use kubectl on the daily bases, it might be better to reference it as standalone app `kjobctl` and not a kubectl plug-in.

Reply:
I'm not sure because on doc said:

> Kjobctl will be provided as a kubectl plugin (and reuse all authentication/configuration methods that are available in kubectl).